tests/test_fields.py

```diff
@@ -1875,26 +1877,26 @@ def test_edit_choices(self):
             field.run_validation(2)
         assert exc_info.value.detail == ['"2" is not a valid choice.']
 
-    def test_integer_choices(self):
-        class ChoiceCase(IntegerChoices):
+    def test_enum_choices(self):
```

Comment:
It would make sense to add this new test in addition to keeping the old `IntegerChoices` test just to ensure we don't have regressions in future versions of Django if they decide to do something special with that class.